apis/src/pages/profile_view.rs

```diff
@@ -10,8 +12,16 @@ struct UsernameParams {
     username: String,
 }
 
+#[derive(Clone, PartialEq)]
+pub enum ProfileGamesView {
+    Playing,
+    Finished,
+}
+#[derive(Clone)]
+pub struct AllUserGames(pub Vec<GameResponse>, pub Vec<GameResponse>);
```

Comment:
Is the idea here for this to be a placeholder until we make a provider like `Games` for this?